packages/flutter/test/material/scaffold_test.dart

```diff
@@ -32,7 +32,7 @@ void main() {
     await tester.pumpWidget(new Directionality(
       textDirection: TextDirection.ltr,
       child: new MediaQuery(
-        data: const MediaQueryData(padding: const EdgeInsets.only(bottom: 100.0)),
```

Comment:
also test that a mediaquery with a padding doesn't mess with the scaffold layout in unexpected ways?

Reply:
And maybe that both can exist and still works ok

Reply:
Should be covered by the last test (line 553ish). I'm okay with adding it on the others for backup though -- wdyt?